mautic/mautic
# Fix DNC report channel link

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:

This fix

`Internal Server Error - PHP Notice - Undefined index: channel_id`



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create do not contact report
3. Add contact email and DNC channel as column
4. Save and check report
5. See error `Internal Server Error - PHP Notice - Undefined index: channel_id`
6. After patch you see correct data
